k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
Gotcha.  That's essentially what I imagined.

I would like uPortal to have an appropriate public registry to push to, whatever that means... some kind of "Docker Central" or an Apereo/uPortal registry.  I think we should work on that.

But for as long as we don't have an appropriate registry, I think...
  - We shouldn't list one in the default configs;  but
  - We should make it easy to specify one, and document (1) how you do that and (2) why you might want to